<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openCvTestClass</class>
 <widget class="QMainWindow" name="openCvTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>openCvTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>481</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>481</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>550</y>
      <width>149</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_cut">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>550</y>
      <width>149</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>cut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>550</y>
      <width>149</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>620</y>
      <width>149</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>620</y>
      <width>149</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_finished">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>620</y>
      <width>149</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>采集</string>
    </property>
   </widget>
   <widget class="QWidget" name="wgt_dev" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>977</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="openCvTest.qrc"/>
 </resources>
 <connections/>
</ui>
